crates/terminal/build.rs

```diff
@@ -0,0 +1,11 @@
+fn main() {
```

Comment:
We have a https://github.com/zed-industries/zed/blob/d32e9f759c289a111f4a104b40f258c00fd4727b/crates/lsp/src/lsp.rs#L667 with the app version.

does this way not work for the case?

Reply:
Thanks for pointing this out, I hadn't noticed this before. Fixed.